explorer/websockethandlers.go

```diff
@@ -21,6 +21,26 @@ import (
 // close an already closed connection.
 var ErrWsClosed = "use of closed network connection"
 
+func trimTxInfo(txs []*TxInfo) []*TrimmedTxInfo {
+	trimmedTxs := make([]*TrimmedTxInfo, 0, len(txs))
+	for _, tx := range txs {
+		voteValid := false
```

Comment:
This is fine, but the style used in our code is usually to declare like `var v someType` when the zero value for `someType` is desired for the value of the new variable `v` (i.e. `var voteValid bool`)